qadence/models/identity_qnn.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+
+import itertools
+from typing import Callable
+
+import numpy as np
+from torch import Tensor
+
+from qadence.backend import BackendConfiguration, BackendName
+from qadence.backends.pytorch_wrapper import DiffMode
+from qadence.blocks import AbstractBlock, chain, kron, tag
+from qadence.blocks.utils import parameters
+from qadence.circuit import QuantumCircuit
+from qadence.measurements import Measurements
+from qadence.models.quantum_model import QuantumModel
+from qadence.operations import CNOT, RX, RY
+
+
+def identity_block(
```

Comment:
I'd suggest moving this to qadence.blocks under the name identity.py

Reply:
Actually `qadence.blocks` is mostly about the infrastructure of the block system :). I would suggest `qadence.constructors` for functions creating specific block compositions that can be generally useful for different algorithms. There is already an `ansatze.py` file there, but it's pretty dedicated to the hardware-efficient ansatz (and thus might need a rename), so this one might be better off in a separate file.

Reply:
Thanks @jpmoutinho, I guess it might make sense to be in qadence.constructors then.

Reply:
Thanks @jpmoutinho, will move it there!